packages/editor-ui/src/views/NodeView.vue

```diff
@@ -601,7 +601,7 @@ export default defineComponent({
 		},
 		runButtonText(): string {
 			if (!this.workflowRunning) {
-				return this.$locale.baseText('nodeView.runButtonText.executeWorkflow');
+				return this.$locale.baseText('nodeView.runButtonText.testWorkflow');
```

Comment:
This is one of the rare times when it would have been probably okay, even better, to keep the same key name.

Reply:
will revert back

Reply:
it's okay, not a big one.